pkgs/tools/misc/wyrd/default.nix

```diff
@@ -32,7 +32,7 @@ stdenv.mkDerivation rec {
     downloadPage = "http://pessimization.com/software/wyrd/";
     license = licenses.gpl2Only;
     maintainers = [ maintainers.prikhi ];
-    platforms = platforms.linux;
+    platforms = platforms.linux ++ platforms.darwin;
```

Comment:
```suggestion
    platforms = platforms.unix;
```

Reply:
Thanks for the suggestion, @DontEatOreo, I've made the change in the latest force-push to this PR branch.